src/python/module/smartredis/dataset.py

```diff
@@ -59,15 +59,16 @@ def _data(self) -> PyDataset:
 
     @staticmethod
     def from_pybind(dataset: PyDataset) -> "Dataset":
-        """Initialize a Dataset object from
-        a PyDataset object
+        """Initialize a Dataset object from a PyDataset object
+
+            Create a new Dataset object using the data and properties 
```

Comment:
Keep text left-justified with block above

Reply:
The tab tells the rst file to make an indent note, the have this format like below, thoughts?